### PR TITLE
Added a new shape category (glow), and new graphic category registered with item IDs that are different from other shpes and come with light value. This allows for better visual effects by modifying the configuration settings based on the item ID in the presence of shader

### PR DESCRIPTION
**Pull request by ImgoodWK**
给一部分斜面添加了单独的分类，用于与正常的斜面区分使用，以此达到通过光影单独配置发光斜面的效果，目前添加一些斜面形状，后续会继续增加别的形状。

### CODE REVIEW

**Comment by Dream-Master** [comment]
@ImgoodWK can you translate the text to englisch so more people know what tgis pr is about?
please run spotless apply

**Comment by bombcar** [comment]
Seems to be adding support for shaders, or the beginning of such support.

**Comment by MCTBL** [comment]
Add a separate category of the ramp, in order to achieve the effect of different shaderpacks through the separate configuration of the luminous ramp.
Now just add some of ramp shapes, will continue to add more other shapes.

> 给一部分斜面添加了单独的分类，用于与正常的斜面区分使用，以此达到通过光影单独配置发光斜面的效果，目前添加一些斜面形状，后续会继续增加别的形状。

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Add a separate category of the ramp, in order to achieve the effect of different shaderpacks through the separate configuration of the luminous ramp.
```
Comment (OneEyeMaker) [inline README.md]: We don't store changelogs in readme file. Put all this text in PR description, if you want to present the list of changes.

File: src/main/java/gcewing/architecture/RenderRoof.java [hunk 2]
```diff
@@ -49,33 +49,63 @@ protected void render() {
...
                 break;
+            case SlopeTileC3SE:
+                renderSlopeC3();
+                break;
             case SlopeTileC4:
                 renderSlopeC4();
                 break;
+            case SlopeTileC4SE:
+                renderSlopeC4();
+                break;
```
Comment (OneEyeMaker) [inline src/main/java/gcewing/architecture/RenderRoof.java]: This switch can be simplified:
```suggestion
            case RoofTile:
            case RoofTileSE:
                renderSlope();
                break;
            case SlopeTileA1:
            case SlopeTileA1SE:
                renderSlopeA1();
                break;
            case SlopeTileA2:
            case SlopeTileA2SE:
                renderSlopeA2();
                break;
            case SlopeTileB1:
            case SlopeTileB1SE:
                renderSlopeB1();
                break;
            case SlopeTileB2:
            case SlopeTileB2SE:
                renderSlopeB2();
                break;
            case SlopeTileB3:
            case SlopeTileB3SE:
                renderSlopeB3();
                break;
            case SlopeTileC1:
            case SlopeTileC1SE:
                renderSlopeC1();
                break;
            case SlopeTileC2:
            case SlopeTileC2SE:
                renderSlopeC2();
                break;
            case SlopeTileC3:
            case SlopeTileC3SE:
                renderSlopeC3();
                break;
            case SlopeTileC4:
            case SlopeTileC4SE:
                renderSlopeC4();
                break;
```

File: src/main/java/gcewing/architecture/SawbenchTE.java [hunk 3]
```diff
@@ -262,6 +280,12 @@ protected void showMaterial() {
 
     protected ItemStack makeResultStack() {
         Shape resultShape = getSelectedShape();
+        boolean shaderEmissive;
+        if (getSelectedPage() == 7) {
+            shaderEmissive = true;
+        } else {
+            shaderEmissive = false;
+        }
```
Comment (OneEyeMaker) [inline src/main/java/gcewing/architecture/SawbenchTE.java]:
```suggestion
        boolean shaderEmissive = getSelectedPage() == 7;
```

File: src/main/resources/assets/architecturecraft/lang/zh_CN.lang [hunk 4]
```diff
@@ -1,5 +1,6 @@
 tile.architecturecraft:sawbench.name=建筑师锯台
 tile.architecturecraft:shape.name=构件
+tile.architecturecraft:shapeSE.name=构件(发光)
```
Comment (OneEyeMaker) [inline src/main/resources/assets/architecturecraft/lang/zh_CN.lang:3]: If you add any localizable text, please make sure to add English (`en_US`) localization (`cause most players play in English).

**Review by OneEyeMaker — CHANGES_REQUESTED** [review]
Nice addition- yet another way to add glowing blocks.
Just some code cleanup is needed.

File: src/main/resources/assets/architecturecraft/lang/zh_CN.lang (hunk 4 above)
Comment (ImgoodWK) [reply src/main/resources/assets/architecturecraft/lang/zh_CN.lang:3]: > If you add any localizable text, please make sure to add English (`en_US`) localization (`cause most players play in English).

Thank you! In fact, this is my first time submitting a PR, and I haven't coded in a long time. I appreciate your suggestions and I will make the necessary modifications when I have the time.

**Review by OneEyeMaker — APPROVED** [review]

**Comment by OneEyeMaker** [comment]
Please, resolve conflict with current master branch, if you can. Thank you.

**Pull request merged**